openassessment/xblock/openassessmentblock.py

```diff
@@ -183,6 +190,18 @@ class OpenAssessmentBlock(
         help="The type of prompt. html or text"
     )
 
+    resubmissions_grace_period_hours = Integer(
+        default=0,
+        scope=Scope.content,
+        help="The number of hours after the submission due date that resubmissions are allowed."
```

Comment:
After the submission due date or after the students' submission? After the submission due date means that resubmissions are allowed after the [problem's due date](https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/exercises_tools/open_response_assessments/CreateORAAssignment.html#configure-deadlines-manually), which is not what the cover letter specifies. Is that correct?

Reply:
Is after the student's submission date, I will correct it.